apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/behandling/SakOgBehandlingService.kt

```diff
@@ -71,10 +74,17 @@ class SakOgBehandlingService(
             )
         }
 
-        val soeskenjustering = beregning.beregningsperioder
-            .any { !it.soeskenFlokk.isNullOrEmpty() }
+        val soeskenjustering = beregning.beregningsperioder.any { !it.soeskenFlokk.isNullOrEmpty() }
+
+        val beloep = if (beregningsperioder.size == 1) beregningsperioder.first().utbetaltBeloep
+        else {
+            val today = LocalDate.now()
+            beregningsperioder.find { it.datoFOM.isBefore(today) && it.datoTOM?.isAfter(today) ?: true }?.utbetaltBeloep!! // ktlint-disable max-line-length
```

Comment:
Denne linja ser litt sketchy ut 😅

Reply:
Hvis jeg forstår logikken rett her, så kunne man kanskje sortert beregningsperiodene etter tid, og velge første - uavhengig av om det er 1 eller flere perioder. Så kan man håndtere tom liste med en exception i steden for `!!`

Reply:
Støttes det her at man kan få to utbetalingsperioder? Feks dersom personen dør i april, men man søker i juni. Da risikerer vi at man får en g-regulering og dermed to utbetalingsperioder fra beregning.

Reply:
Har endret det til å hente ut fra den nyeste perioden og lagt til Todo for håndtering av g-regulering. Det er ikke behov for å håndtere g-regulering enda.